### DISCUSSION TITLE
SpeedCrunch

### DISCUSSION THREAD

**Discussion by Zazucki** [catppuccin/catppuccin#2253] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://heldercorreia.bitbucket.io/speedcrunch/advanced/colorschemeformat.html

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closing as further discussion should continue in #2254
